### DISCUSSION TITLE
Is there a way to Toggle On/Off Android System Bar?

### DISCUSSION THREAD

**Discussion by Gullzway** [AndreyPavlenko/Fermata#535] [General]
Sorry if I missed this but I can't find it in settings anywhere. 
Is there a way to toggle off/on the Android auto system bar to get actual full screen?
On my Ford 8-in screen it takes up a inch of the entire bottom screen. 

On my jailbroken iPhone the tweak AIRAW allows you to select certain apps to enable full screen. You can then press a toggle on the car play bar and it disappears. You just tap in the corner again and it reappears.

https://chariz.com/buy/airaw

**Comment by AndreyPavlenko** [comment]
As I remember, there were some tools like Substratum. But root is required and I haven't used it for a long time, not sure if it still works.

**Reply by Gullzway** [reply]
Ah, Ok Thanks.
